website/docs/docs/dbt-versions/core-upgrade/04-upgrading-to-v1.11.md

```diff
@@ -52,6 +52,12 @@ Coming soon to dbt Core v1.11, deprecation warnings from JSON Schema validation
 
 These warnings help you proactively identify and update deprecated configurations (such as misspelled config keys, deprecated properties, or incorrect data types).
 
+## Adapter-specific features and functionalities
+
+### Snowflake
+
+- The Snowflake adapter supports basic table materialization on Iceberg tables registered in a Glue catalog through a [catalog-linked database](https://docs.snowflake.com/en/user-guide/tables-iceberg-catalog-linked-database#label-catalog-linked-db-create). For more information, see [Table materialization in dbt-snowflake](/docs/mesh/iceberg/snowflake-iceberg-support#table-materialization-in-dbt-snowflake).
```

Comment:
the link also doesn't take me anywhere (maybe its jsut me!) but should it be this section instead: https://docs-getdbt-com-git-upgrade-guide-updates-dbt-labs.vercel.app/docs/mesh/iceberg/snowflake-iceberg-support#external-catalogs?